cmd/gendeepcopy/deep_copy.go

```diff
@@ -104,7 +104,7 @@ func main() {
 	} else {
 		pkgname = gv.Group
 	}
-	if len(gv.Version) != 0 {
```

Comment:
gruntime is worse - but I can't stop using it.  That's why this one is
not "runtime" - stdlib takes precedence.